cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -439,6 +441,45 @@ function(ly_test_impact_write_config_file CONFIG_TEMPLATE_FILE BIN_DIR)
     message(DEBUG "Test impact framework post steps complete")
 endfunction()
 
+function(ly_test_impact_write_pytest_file)
+    set(config_array "")
```

Comment:
Would be useful to see a comment for each of these sections explaining what is going on.